.sbtrc

```diff
@@ -0,0 +1,5 @@
+# Test run control
+alias test=; testOnly -l IntegrationTest
+alias testDB=; testOnly -t IntegrationTest
```

Comment:
Shouldn't this be limited to `database` module?

Reply:
After removing `testDB` alias this looks no more valid.

Reply:
Alias are now used use only for the database module - valid for ScalaTest.
The server module has different solution valid for the Zio framework.

Reply:
In call prepared solution. TODO